app/assets/javascripts/comments.js

```diff
@@ -1,151 +1,319 @@
-function initCommentOptions(scrollableContainer, useParentOffset) {
-  useParentOffset = (typeof useParentOffset !== 'undefined') ? useParentOffset : true;
-  scrollCommentOptions($(".dropdown-comment"), useParentOffset);
-
-  // Reposition dropdown to the left
-  // (only do this when using parent offset)
-  if (useParentOffset) {
-    $(document).on("shown.bs.dropdown", ".dropdown-comment", function() {
-      var $el = $(this);
-      var menu = $el.find(".dropdown-menu");
-      var leftPos = $el.offset().left;
-      if (leftPos + menu.width() > $(window).width()) {
-        menu.offset({ left: leftPos - menu.width() });
+var Comments = (function() {
```

Comment:
I would remove _var Comments =_ and youst add comment saying what it represents.